src/components/menu/menu/menu.component.html

```diff
@@ -4,6 +4,7 @@
         class="ux-menu"
         [class.ux-sub-menu]="_isSubMenu"
         [class.ux-menu-placement-top]="placement === 'top'"
+        [attr.role]="'menu'"
```

Comment:
```suggestion
        role="menu"
```

No need for a binding here as it is not a dynamic value